C0T0/Classes/ApiService/ApiService.swift

```diff
@@ -44,12 +46,11 @@ public class ApiService {
             urlRequestBuilder: StandardURLRequestBuilder(host:configuration.host),
             urlRequestSender: StandardURLRequestSender(urlSession: URLSession.shared),
             converter: StandardURLRequestSenderErrorConverter(),
-            configuration: ApiDataSourceConfiguration()
+            configuration: ApiDataSourceConfiguration(loggingEnabled: configuration.loggingEnabled)
```

Comment:
We should pass a `keyDecodingStrategy` here as well, it is not used anywhere right now.